components/app/Footer.tsx

```diff
@@ -0,0 +1,102 @@
+import Link from 'next/link'
+
+function Content() {
+  return (
+    <section>
+      <h2 className="uppercase text-sm font-semibold text-white text-opacity-75">
+        Content
+      </h2>
+      <ul className="text-sm text-white text-opacity-50">
+        <li>
+          <Link href="/browse">
+            <a>Browse</a>
+          </Link>
+        </li>
+        <li>
+          <Link href="/courses">
+            <a>Courses</a>
+          </Link>
+        </li>
+        <li>
+          <Link href="/lessons">
+            <a>Lessons</a>
+          </Link>
+        </li>
+        <li>
+          <Link href="/podcasts">
+            <a>Podcasts</a>
+          </Link>
+        </li>
+      </ul>
+    </section>
+  )
+}
+function About() {
+  return (
+    <section>
+      <h3 className="uppercase text-sm font-semibold text-white text-opacity-75">
```

Comment:
Was just going for "close enough" with these. egghead-rails uses a custom "opacity-80" and such, but I didn't want to commit to custom styles until we agree upon how all the global styles/vars work.

Reply:
We should be good with default tailwind styles for foreseeable future.